earth_enterprise/rpms/build.gradle

```diff
@@ -79,6 +79,16 @@ def gee_long_version_txt = "${src_dir}${native_dir()}/gee_long_version.txt"
 
 def rpmPlatformString = org.opengee.os.Platform.rpmPlatformString
 
+def osId = org.opengee.os.Platform.osId
+
+def packageDep
```

Comment:
Minor: I don't think we need a new variable just to change one dependency.  It could just be:

```Groovy
task osPackage(dependsOn: org.opengee.os.Platform.osId == "ubuntu" ?  "openGeeDebs" : "openGeeRpms")
```